src/events/guildCreate.ts

```diff
@@ -0,0 +1,34 @@
+import { Guild, GuildMember } from 'discord.js';
+import { Event } from 'klasa';
+
+export default class extends Event {
+	public run(guild: Guild) {
+		const membersInVoice: GuildMember[] = [];
+
+		// Cache all tags and prepare for sweep
+		for (const member of guild.members.cache.values()) {
+			this.client.userTags.create(member.user);
+			guild.memberTags.create(member);
+
+			if (member.voice.channelID) membersInVoice.push(member);
+		}
+
+		const { me } = guild;
```

Comment:
@kyranet why do we have this anyway btw? Seems a bit useless considering there's only 1 usage of `guild.me` (like 29)